spec/game_runner_spec.rb

```diff
@@ -16,11 +16,29 @@
       GRID
 
       stdout = StringIO.new
-      game_start = GameRunner.new(stdout)
+      stdin = StringIO.new
+      game_start = GameRunner.new(stdout, stdin)
 
       game_start.run
 
-      expect(stdout.string).to eq(grid_template)
+      expect(stdout.string).to include(grid_template)
+    end
+
+    context "after rendering the grid, the user is prompted for input" do
+      it "it quits when a user enters a good coordinate" do
+        stdout = StringIO.new
+        stdin = StringIO.new('A2')
+        game_start = GameRunner.new(stdout, stdin)
+        grid_coordinates = ["A1", "A2", "A3", "B1", "B2", "B3", "C1", "C2", "C3"]
+
+        game_start.run
+
+        expect(stdout.string).to include("Enter your move >")
+        expect(game_start.grid_coordinates).to match(grid_coordinates)
+        expect(game_start.grid_coordinates).to include(stdin.string)
```

Comment:
`game_start.grid_coordinates` doesn't exist yet, of course, but I was looking to make it explicit what I was trying to implement. What do you think?